release-content/0.14/release-notes/10973_Contextually_clearing_gizmos.md

```diff
@@ -1 +1,3 @@
-TODO
+Gizmos are drawn via an immediate mode API. This means that every **update** you draw all gizmos you want to display. Only those will be shown. Previously, update referred to "once every time the `Main` schedule runs". This matches the frame rate, so it usually works great! But when you try to draw gizmos during `FixedMain`, they will flicker or be rendered multiple times. In Bevy 0.14, this now just works!
+
+This is extendable for custom schedules: Instead of a single storage, there now are multiple [storages](https://dev-docs.bevyengine.org/bevy/gizmos/gizmos/struct.GizmoStorage.html) differentiated by a context type parameter. You can also set a type parameter on the [`Gizmos`](https://dev-docs.bevyengine.org/bevy/gizmos/gizmos/struct.Gizmos.html) system param to choose what storage to write to. You choose when storages you add get drawn or cleared: Any gizmos in the default storage (the `()` context) during the `Last` schedule will be shown.
```

Comment:
```suggestion
This can be extended for use with custom schedules. Instead of a single storage, there now are multiple [storages](https://dev-docs.bevyengine.org/bevy/gizmos/gizmos/struct.GizmoStorage.html) differentiated by a context type parameter. You can also set a type parameter on the [`Gizmos`](https://dev-docs.bevyengine.org/bevy/gizmos/gizmos/struct.Gizmos.html) system param to choose what storage to write to. You choose when storages you add get drawn or cleared: Any gizmos in the default storage (the `()` context) during the `Last` schedule will be shown.
```